api/src/start.ts

```diff
@@ -17,7 +19,21 @@ export default async function start(): Promise<void> {
 	const port = env.PORT;
 
 	server
-		.listen(port, () => {
+		.listen(port, async () => {
+			let update = null;
+
+			try {
+				update = await checkForUpdate(pkg);
+			} catch (err) {
+				// eslint-disable-next-line no-console
+				console.log(err);
+			}
+
+			if (update) {
+				logger.warn(`New Version Available (${update.latest})\nYou can update by running: npx ${pkg.name}@latest [command]
```

Comment:
```suggestion
				logger.warn(`Update available: ${pkg.version} -> ${update.latest}`);
```

Seeing that Directus can run in all sorts of environments (Docker container, npm directly, hosted services, etc), we can't know for sure how to update Directus in your current environment